examples/deck.gl/3d-tiles/app.js

```diff
@@ -200,8 +200,15 @@ export default class App extends PureComponent {
   }
 
   _renderControlPanel() {
-    const {examplesByCategory, category, name, viewState, selectedMapStyle} = this.state;
-    const {attributions} = this.state;
+    const {
```

Comment:
Nit: I realize this is just automatic prettier change, but this is now adding too many lines to the example. 

Our goal should be to try to keep `app.js` as compact as we can.

Reply:
will do a followup PR